src/data/analyse_en.json

```diff
@@ -1,4 +1,314 @@
+        "qr_redeemable_cumulated": "QR codes",
+        "teletan_redeemable_cumulated": "teleTANs",
+        "qr_not_redeemed_cumulated": "Not shared",
+        "qr_redeemed_cumulated": "Shared",
+        "teletan_not_redeemed_cumulated": "Not shared",
+        "teletan_redeemed_cumulated": "Shared",
+
+        "qr_teletan_redeemable_daily": "Total",
+        "qr_redeemable_daily": "QR codes",
+        "teletan_redeemable_daily": "teleTANs",
+        "qr_not_redeemed_daily": "Not shared",
+        "qr_redeemed_daily": "Shared",
+        "teletan_not_redeemed_daily": "Not shared",
+        "teletan_redeemed_daily": "Shared",
+
+        "qr_teletan_redeemable_7days_sum": "Total",
+        "qr_redeemable_7days_sum": "QR codes",
+        "teletan_redeemable_7days_sum": "teleTANs",
+        "qr_not_redeemed_7days_sum": "Not shared",
+        "qr_redeemed_7days_sum": "Shared",
+        "teletan_not_redeemed_7days_sum": "Not shared",
+        "teletan_redeemed_7days_sum": "Shared",
+
+        "qr_teletan_redeemable_7days_avg": "Total",
+        "qr_redeemable_7days_avg": "QR codes",
+        "teletan_redeemable_7days_avg": "teleTANs",
+        "qr_not_redeemed_7days_avg": "Not shared",
+        "qr_redeemed_7days_avg": "Shared",
+        "teletan_not_redeemed_7days_avg": "Not shared",
+        "teletan_redeemed_7days_avg": "Shared",
+
+
+
+
+        "ppa_total_warnings_cumulated": "Total",
+        "ppa_risk_red_cumulated": "Red risk",
+        "ppa_risk_green_cumulated": "Green risk",
+
+        "ppa_total_warnings_daily": "Total",
+        "ppa_risk_red_daily": "Red risk",
+        "ppa_risk_green_daily": "Green risk",
+
+        "ppa_total_warnings_7days_sum": "Total",
+        "ppa_risk_red_7days_sum": "Red risk",
+        "ppa_risk_green_7days_sum": "Green risk",
+
+        "ppa_total_warnings_7days_avg": "Total",
+        "ppa_risk_red_7days_avg": "Red risk",
+        "ppa_risk_green_7days_avg": "Green risk"
+
+
+
+
+    },
+    "moreHeadline": "Further dashboards",
+    "moreLinks": [
+    	{
+    		"title": "RKI: COVID-19 Dashboard (GER)",
+            "desc": "Daily updated information on the incidence of infections in Germany by county and federal state.",
+    		"link": "https://experience.arcgis.com/experience/478220a4c454480e823b17327b2bf1d4"
+    	},
+    	{
+    		"title": "RKI: Covid-19 Trends (GER)",
+            "desc": "Overview of the trends of the epidemic situation in Germany.",
+    		"link": "https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Situationsberichte/COVID-19-Trends/COVID-19-Trends.html?__blob=publicationFile#/home"
+    	},
+    	{
+    		"title": "BMG: Impfdashboard (EN)",
+            "desc": "Workdaily updated information on vaccination status and progress in Germany, from the website 'together against corona' from the German federal ministry of health (BMG).",
+    		"link": "https://impfdashboard.de/en/"
+    	},
+    	{
+    		"title": "WHO: COVID 19 Dashboard (EN)",
```

Comment:
Suggest
"COVID 19" => "COVID-19"

to match the text on the website https://covid19.who.int/ itself.